lib/services/awsAdapter.js

```diff
@@ -109,15 +108,12 @@ function AWSAdapter (options) {
 		});
 	};
 
-	this.beginPolling = function (result) {
+	this.updatePollingStatus = function (result) {
 		var self = this;
 		var instanceIPAddress;
 
-		return sshAdapter.SshPolling(result.instanceID)
-		.then(function () {
-			serverLog([ "info", "AWSAdapter", "SshPolling" ], "Ssh Polling is successful");
-			return self.getPublicIPAddress(result.instanceID);
-		})
+		serverLog([ "info", "AWSAdapter", "SshPolling" ], "Ssh Polling is successful");
```

Comment:
Since we're revisiting this code, I think it would be helpful to add the instance id and ip address to this log. Otherwise, it doesn't seem too useful in a live environment ("which of our concurrent instance polls was successful?")